scrapers/parse_scrape_output.py

```diff
@@ -196,7 +201,15 @@ def parse_date(d):
       continue
     assert False, f"Unknown data on line {i}: {l}"
 
-  print("{:2} {:<16} {:>7} {:>7} OK {} {}".format(abbr, date, cases, deaths if not deaths is None else "-", scrape_time, ", ".join(url_sources)))
+  if date and cases:
+    print("{:2} {:<16} {:>7} {:>7} OK {} {}".format(abbr, date, cases, deaths if not deaths is None else "-", scrape_time, ", ".join(url_sources)))
+  else:
+    errs = []
+    if not date:
+      errs.append("Missing date")
+    if not cases:
+      errs.append("Missing cases")
+    print("{:2} {:<16} {:>7} {:>7} FAILED {} {} {}".format(abbr, date if date else "-", cases if not cases is None else "-", deaths if not deaths is None else "-", scrape_time if not scrape_time is None else "-", ". ".join(errs), ", ".join(url_sources)))
 
```

Comment:
In the error case, you should raise an Exception so that the `except` block is reached and sys.exit(1) is called.

Reply:
This is exactly what I don't want.

I want to handle the error, print a message, and continue without a program without an error.